frontend-react/src/utils/misc.ts

```diff
@@ -175,3 +175,27 @@ export const parseFileLocation = (
         fileName,
     };
 };
+
+export const removeHTMLFromString = (input: string) => {
+    return input.replace(/<(.|\n)*?>/g, "");
+};
+
+export const convertStringToSortableRepresentation = (
+    input: string | undefined
+): string | number => {
+    let convertedContent = "";
+    if (!input) return convertedContent;
+
+    // Check if input is a date, then we could leave it as is,
+    // Or convert to 100% safe JS Date String
+    if (!!Date.parse(input)) {
+        convertedContent = new Date(input).toString();
```

Comment:
If it's a date, why couldn't we just compare the number of milliseconds via `getTime()`?

Reply:
The expected return value should be a `string`.